Scripts/script-ReadPDFFile.yml

```diff
@@ -58,6 +58,8 @@ script: |-
           except PDFEncryptionError:
               mark_suspicious('Possibly encrypted file')
               sys.exit(0)
+          except TypeError:
```

Comment:
Can we try to add some other identification as well? this seems a too broad exception type for this error.
And maybe we should mark it as suspicious as we are doing for the other exceptions